src/iscsi.pyx

```diff
@@ -6,18 +6,50 @@
 
 from libc.stdlib cimport calloc
 
+# Not needed yet (see struct scsi_sense)
+# cdef extern from "sys/types.h":
+#    ctypedef unsigned short uint16_t
+#    ctypedef unsigned long long uint64_t
 
 cdef extern from "iscsi/scsi-lowlevel.h":
     cpdef enum scsi_xfer_dir:
         SCSI_XFER_NONE
         SCSI_XFER_READ
         SCSI_XFER_WRITE
 
+    cpdef enum scsi_sense_key:
+        SCSI_SENSE_NO_SENSE
+        SCSI_SENSE_RECOVERED_ERROR
+        SCSI_SENSE_NOT_READY
+        SCSI_SENSE_MEDIUM_ERROR
+        SCSI_SENSE_HARDWARE_ERROR
+        SCSI_SENSE_ILLEGAL_REQUEST
+        SCSI_SENSE_UNIT_ATTENTION
+        SCSI_SENSE_DATA_PROTECTION
+        SCSI_SENSE_BLANK_CHECK
+        SCSI_SENSE_VENDOR_SPECIFIC
+        SCSI_SENSE_COPY_ABORTED
+        SCSI_SENSE_COMMAND_ABORTED
+        SCSI_SENSE_OBSOLETE_ERROR_CODE
+        SCSI_SENSE_OVERFLOW_COMMAND
+        SCSI_SENSE_MISCOMPARE
+
     cdef struct scsi_task:
         pass
 
     cdef struct scsi_sense:
-        pass
+        # The following are present in libiscsi 1.13.0
+        unsigned char       error_type
+        int                 key
```

Comment:
I was going to say I'm a bit uncomfortable not using the std types for these, but Cython only needs to know about whatever field you want to fetch, so I would rather avoid documenting whatever is there that we don't access instead.

Reply:
Ok, I have removed the additional documentation.